client/src/components/SizeDropDown.jsx

```diff
@@ -60,7 +60,11 @@ class SizeDropDown extends React.Component {
         <DropDownHeader selectedSize={this.state.selectedSize} toggleDropDown={this.toggleDropDown}/>
         <DropDownList dropDownOpen={this.state.dropDownOpen}>
           {this.props.sizes.map((size) => {
-            return <DropDownItem key={size} value={size} onClick={this.handleDropDownSubmit}>{size}</DropDownItem>
+            if (size.stock === 0) {
+              return <DropDownItem key={size.size} value={size.size} onClick={this.handleDropDownSubmit}>{size.size}(out of stock)</DropDownItem>
```

Comment:
size.size is a little odd sounding here, but looks good to me otherwise.

Reply:
The size object has two properties: size and stock. I couldn't think of a good name at the time, maybe size.sizeCode or itemSize.size? 

The size object is also nested inside a color object, so I have to keep that context in mind when naming these properties.

color.itemSize.size vs. color.size.sizeCode, idk which one will make it easier to read in the future.

Reply:
Yeah that's tough, maybe the first object could be named something like "sizeInfo"?

Reply:
I agree here, I actually did a double take when I saw "size.size". I think sizeInfo will make things a little less confusing.